README.md

```diff
@@ -29,9 +31,10 @@ After installation, make sure to commit the `.ddev` directory to version control
 
 ## Usage
 
-- Your project is now ready to run FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://gitlab.com/weitzman/drupal-test-traits) (DTT). All these types are tested in this repo. Some examples to try:
-  - FunctionalJavascript:
+- Your project is now ready to run [Functional](https://mglaman.dev/blog/do-you-need-functional-test), FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://gitlab.com/weitzman/drupal-test-traits) (DTT). All these types are tested in this repo. Some examples to try:
+  - Functional and FunctionalJavascript:
     - Ensure you have the `drupal/core-dev` Composer package or equivalent.
+    - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit -c ./core/phpunit.xml.dist ./core/modules/migrate/tests/src/Functional/process/DownloadFunctionalTest.php"`
     - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit -c ./core/phpunit.xml.dist ./core/modules/system/tests/src/FunctionalJavascript/FrameworkTest.php"`
   - Nightwatch
     - `ddev exec -d /var/www/html/web/core yarn install` (do this once)
```

Comment:
Lets break this into three bullets instead of grouping Functional and FunctionalJavascript